### ISSUE TITLE
Default checkout after creating branch

### ISSUE THREAD

**Issue by aarondandy** [ForkIssues/TrackerWin#623]
When creating a branch using the "New Branch" button my workflow would be sped up a tiny bit if the checkbox to "Check out after create" was checked by default. There may be those that disagree with me on this and there may be other that agree with me. Maybe a setting for the default or remembering the last selection would be a compromise.

**Comment by aarondandy** [comment]
Also, the text "All uncommitted changes will be moved to the new branch" only really applies if you also "Check out after create" right?

**Comment by aarondandy** [comment]
Today it was default checked for me, I can't tell if it already does this behavior or not. If I does maybe I should rephrase my suggestion to be that the initial state of the checkbox should be checked after the first run of the application.

**Comment by davidaue** [comment]
The current behavior seems to be that it persists the last state, even when restarting the application. This would seem to be ideal for most users.

**Comment by dracan** [comment]
> The current behavior seems to be that it persists the last state, even when restarting the application. This would seem to be ideal for most users.

Yep, that's the current behaviour. I disagree though that it's ideal - I personally find it really frustrating. Just raised an issue about it here: #2702.
